initfiles/sbin/hpcc_setenv.in

```diff
@@ -81,6 +81,7 @@ IFS="${OIFS}"
 PATH_PREFIX=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^path *= *" | sed -e 's/^path *= *//'`
 
 export PID=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^pid *= *" | sed -e 's/^pid *= *//'`
+export LD_PRELOAD=`cat ${HPCC_CONFIG} | sed -n "/\[${SECTION}\]/,/\[/p" | grep "^java *= *" | sed -e 's/^java *= *//'`:$LD_PRELOAD
```

Comment:
Should we test first and then set (or add to) LD_PRELOAD only if this is not empty ?
Probably does not matter but don't see a need to force LD_PRELOAD always, even if empty.
Something like:
jvm_preload=$(cat ...)
if [ -n "${jvm_preload}" ] ; then
  export LD_PRELOAD=${jvm_preload}:${LD_PRELOAD}
fi